lib/scrabble.rb

```diff
@@ -1,7 +1,31 @@
 class Scrabble
 
   def score(word)
-    1
+    word = usable_word(word)
+    single_multipliers = Array.new(word.length, 1)
+    score_with_multipliers(word, single_multipliers)
+  end
+
+  def score_with_multipliers(word, letter_multiplers, word_multiplier = 1)
+    word = usable_word(word)
+
+    score = word.each_char.with_index.reduce(0) do |points, (letter, index)|
+      points + letter_score(letter) * letter_multiplers[index]
+    end
+
+    score * word_multiplier
+  end
+
+  def usable_word(word)
+    if word.is_a?(String)
+      word.upcase
+    else
```

Comment:
I want to give you some quick context about this kind of error handling. There's a concept called failing silently. It's considered kind of a UX faux pas. You'd rather report back to the user that their input was invalid, or if it's a developer using your code, end execution with a well described exception.

We barely mention this or really much error handling at Turing, but since you're starting to invent your own, I figured I'd mention it.